recipes/matrix-api-async/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "matrix-api-async" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8361fd8ffabce911b2f7a43bfdce1b2d7dad8d9a0ff396401f5951109943bd75
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - matrix_client
+    - python >=3.5
+
+test:
+  imports:
+    - matrix_api_async
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Cadair/matrix_api_async
+  summary: An asyncio wrapper of matrix_client.api
+  license: MIT
+  # License manually packaged. See: https://github.com/Cadair/matrix_api_async/issues/1
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - synapticarbors
```

I guess this is mostly my fault lol

```suggestion
    - synapticarbors
    - cadair
```